### ISSUE TITLE
Sugest an example

### ISSUE THREAD

**Issue by VirusTermico** [expo/examples#397]
Firstily congrats for such great work. Those example really help alot.
My suggest it´s based in fact that sometimes we want combine multi libs for example start a project with taiwindcss and bottom tabs.
It woud be awesome if you could provide a project generator similar to springio where people specify the dependecies  and a project it´s generate.

**Comment by Pfed-prog** [comment]
yes, that would be really great. I have trouble installing tailwind
